### DISCUSSION TITLE
Unable to download template from Github

### DISCUSSION THREAD

**Discussion by AmirAskariTHG** [vercel/turborepo#6346] [Help]
Hi, on a MacOs system when I'm trying to run  `npx create-turbo@latest monorepo` I'm getting these errors:

>>> TURBOREPO

>>> Welcome to Turborepo! Let's get you set up with a new codebase.

? Which package manager do you want to use? yarn workspaces

Downloading files. This might take a moment.

>>> Unable to download template from Github
>>> unable to get local issuer certificate

I've tried to download cert files from its github url and put into the cert folders and keychain certs but still not working. Please could you help me?

**Comment by mehulkar** [comment]
@AmirAskariTHG I would recommend searching the web for the error "unable to get local issuer certificate" and troubleshooting your setup that way. For example the first result I see is [this StackOverflow answer](https://stackoverflow.com/questions/23885449/unable-to-resolve-unable-to-get-local-issuer-certificate-using-git-on-windows) about some additional setup on Windows (if that's what you're using). You could try coming into [our Discord community](https://turbo.build/discord) to see if anyone can help out as well.
